gravitee-apim-console-webui/src/entities/management-api-v2/api/v4/healthCheck.fixture.ts

```diff
@@ -32,30 +32,21 @@ export function fakeApiHealthResponseTimeOvertime(attribute?: Partial<ApiHealthR
   };
 }
 
-export function fakeApiHealthAvailability(attribute?: Partial<ApiAvailability>): ApiAvailability {
```

Comment:
we keep this functions with attribute paramiters so we can manipulate result of test data. Its a pattern here.

Reply:
done 👍  however, I still don't get why we use this pattern here if we don't make use of it 🤔

Reply:
cool, I will use it :-)